### DISCUSSION TITLE
Log compliance information in the manager log

### DISCUSSION THREAD

**Discussion by shlomitk1** [fybrik/fybrik#480] [Ideas]
We need to log some information about the data flow:
- The data origin
- Required governance actions
- What transformations have been performed and where (on which cluster the modules ran)
- Buckets that have been allocated during the flow
   - bucket names 
   - regions where the buckets have been allocated
   - buckets that have been deleted
   - persistent buckets 
- New assets that have been added to the catalog.

Did I miss anything? As the first step, this information will be logged in the manager pod (perhaps, in a separate file in the pod?)

**Comment by simanadler** [comment]
@shlomitk1 You are correct, and what you describe is known as lineage.  We need to integrate with existing lineage solutions - both open source and proprietary.  Short term we should make sure that all of the information you mentioned is being written to our logs.  Longer term, we have on the roadmap an item to address visibility / lineage.
